README.md

```diff
@@ -1,2 +1,66 @@
-# to-do-list-app
-This app enables you to create your own todos on different projects. Used tools and languages: webpack, JS, Bootstrap.
+![](https://img.shields.io/badge/Microverse-blueviolet) 
+# ToDo List App
+
+This Application will help you create your projects and your tasks on them every day. We used a webpack to sort JS files when building it. You can see JS ES6 features like closure, arrow function, eventListener and so on.
+
+
+### Built With
+
+![HTML](https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white)
+![CSS](https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white)
+![BOOTSTRAP](https://img.shields.io/badge/bootstrap%20-%23563D7C.svg?&style=for-the-badge&logo=bootstrap&logoColor=white) 
+![JAVASCRIPT](https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E")
+
+## Getting Started
+
+## To get local copy up and running follow these simple example steps
+
+```
+git clone https://github.com/AbdumurodovaZulfizar/to-do-list-app.git
+```
+
+```
+cd to-do-list-app
+```
+### Prerequisites
+
+- Node
+- Webpack
+
+### Setup
+- run `npm install` to install all required node modules
+- if you vant to see live version on your local machine run `npm run build` and open live server
```

Comment:
- [ ] {REQUIRED] Please use tools like grammarly in order to help you with your readme as it is the first thing users check. Here for example, chenge `vant` to `want`